src/components/Rocket/RocketsCard.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/prop-types */
```

Comment:
You can import and use Prop-Types instead of ignoring them, they are already installed

Reply:
I keep having this issue with prop-types. could you show me  how it is done when we get together this morning?